control/navigation.hpp

```diff
@@ -203,8 +203,8 @@ class Navigation
     uint32_t last_update;                               ///< The time of the last navigation update in ms
     uint32_t loop_count;                                ///< A counter for sending MAVLink messages at a lower rate than the function
     control_command_t& controls_nav;                    ///< The pointer to the navigation control structure
```

Comment:
it is not pointer anymore.
we could change it to reference in the comment, isn't it ?